joomla/joomla-cms
# [4.0] Dependency and build script tweaks

Pull Request for Issue # .

### Summary of Changes

1. Remove `uglifycss` and use postcss/cssnano like we do in other areas of the build scripts
2. Remove the promised based `recursive-readdir` dependency and use a small asynchronous custom function to recursively read directories.
3. Update NodeJS + NPM dependencies to latest LTS
4. Update some dev dependencies to their respective latest version

Still room for improvements but this is a start

### Testing Instructions

1. Run `npm i`
2. Ensure there are no errors in the terminal when building

@dgrammatiko thoughts on this?
